Sources/App/Views/ValidateSPIManifest/ValidateSPIManifest+View.swift

```diff
@@ -66,7 +66,8 @@ extension ValidateSPIManifest {
                                         .name("manifest"),
                                         .autofocus(true),
                                         .rows(15),
-                                        .text(model.manifest)
+                                        .text(model.manifest),
+                                        .attribute(named: "maxlength", value: "1000")
```

Comment:
Could also base this on https://swiftpackageindex.com/swiftpackageindex/spimanifest/1.0.0/documentation/spimanifest/manifest/maxbytesize